src/components/common/BlurBackground.tsx

```diff
@@ -6,13 +6,21 @@ import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 
 const isAndroid = Platform.OS === 'android'
 
-export const BlurBackground = (props: { roundCorners?: boolean }) => {
-  const { roundCorners = false } = props
+export const BlurBackground = (props: {
+  /** If the parent requires rounded corners, this prop must be set true in
+   * order to ensure parity between iOS/Android */
+  noRoundedCorners?: boolean
```

Comment:
Let's go with rounded as an option instead of rounded by default. Or add a `RoundedBlurBackground` element which can be used in rounded components.